### ISSUE TITLE
⭐FEAT: Footer icons dont support dark/light mode.

### ISSUE THREAD

**Issue by ArslanYM** [ArslanYM/StarterHive#215]
# TODO:
- Add different default color for footer icons. 
- Dynamic for dark/light mode.
![footer](https://github.com/ArslanYM/StarterHive/assets/104521101/248589b4-44a8-4dd8-9dcf-318234441c82)

**Comment by AniketSaraswat112** [comment]
Hi Arsalan, I would like to work on this issue, pls assign this to me.

**Comment by ArslanYM** [comment]
@AniketSaraswat112 Sure thing, Assigned to you. Dont forger to star this repo.

**Comment by AniketSaraswat112** [comment]
HI ArslanYM, I was thinking to go with #23272A color on light theme and with "white" on dark theme, Attaching the screenshots of both, please review and confirm whether I should go with these colors.

**Light Theme** : 

![image](https://github.com/ArslanYM/StarterHive/assets/56517011/e1090c6d-c86a-4cde-a871-3f08eef2eda1)

**Dark Theme**:

![image](https://github.com/ArslanYM/StarterHive/assets/56517011/5d9e4218-522a-4b3b-867b-f1c8952722ae)

**Comment by ArslanYM** [comment]
> HI ArslanYM, I was thinking to go with #23272A color on light theme and with "white" on dark theme, Attaching the screenshots of both, please review and confirm whether I should go with these colors.
> 
> 
> 
> **Light Theme** : 
> 
> 
> 
> ![image](https://github.com/ArslanYM/StarterHive/assets/56517011/e1090c6d-c86a-4cde-a871-3f08eef2eda1)
> 
> 
> 
> **Dark Theme**:
> 
> 
> 
> ![image](https://github.com/ArslanYM/StarterHive/assets/56517011/5d9e4218-522a-4b3b-867b-f1c8952722ae)
> 
> 

Looks pretty neat. Id love to see some other ideas as well. Some hover effects or some good color combos with the theme. Do your thing.

**Comment by ArslanYM** [comment]
Looks great @AniketSaraswat112 . However, Can we keep the icon colors default for dark mode , and add better colors for light mode. Also the animation is not necessary at this point.

**Comment by AniketSaraswat112** [comment]
Hi @ArslanYM I have kept the default color for dark theme, for light mode I thought of using neon colors for the icons as attached below: 
![image](https://github.com/ArslanYM/StarterHive/assets/56517011/8b1dae7c-3dab-4d4f-8087-41753f2d241d)

I have used the below color scheme: 
![image](https://github.com/ArslanYM/StarterHive/assets/56517011/bab406a8-3e02-46da-96bb-b4f351bb3e68)

Also we can have the black theme for light mode.
Please review this.

**Comment by ArslanYM** [comment]
Looks sick, Would love to the video, Take a screen recording.

**Comment by AniketSaraswat112** [comment]
> Looks sick, Would love to the video, Take a screen recording.

Yeah sure below are the two recordings of the colors that I propose, I also thought of some animation that will look good added them in the first recording while the second one is without animation as you suggested.
Please let me know which colors should I apply with light theme while I have kept the default colors for dark theme, and whether I should keep the animation or remove it:

1) 

https://github.com/ArslanYM/StarterHive/assets/56517011/d9b96f86-b785-4199-ad37-cde839c191ab

2)

https://github.com/ArslanYM/StarterHive/assets/56517011/84f905b6-6ff2-44b7-a099-adc9fb7df8a6

**Comment by ArslanYM** [comment]
@AniketSaraswat112 why havent you raised a PR yet?

**Comment by AniketSaraswat112** [comment]
Hi, @ArslanYM you haven't confirmed the changes, I proposed two changes in my previous reply, please confirm which one should I go ahead with and then I'll create a PR.

**Comment by ArslanYM** [comment]
Go for the first one, but the color for light mode must be changed.
